run-cppcheck.bat

```diff
@@ -38,9 +38,18 @@ if exist "%CPPCHECK_OUT%" (
 	del %CPPCHECK_OUT%
 )
 
+set CPPCHECK_PARAMS=
+set CPPCHECK_PARAMS=%CPPCHECK_PARAMS% --force
+set CPPCHECK_PARAMS=%CPPCHECK_PARAMS% --enable=all
+set CPPCHECK_PARAMS=%CPPCHECK_PARAMS% --xml
+set CPPCHECK_PARAMS=%CPPCHECK_PARAMS% --platform=%CPPCHECK_PLATFORM%
+set CPPCHECK_PARAMS=%CPPCHECK_PARAMS% --output-file=%CPPCHECK_OUT%
```

Comment:
もともと `2> %CPPCHECK_OUT%` だったものをこう変えて同じ意味になるのか疑問です。
[マニュアル](http://cppcheck.sourceforge.net/)を見た感じ、元の書き方を推奨してる？
しかも `--output-file` というオプションの記載がないような。

Reply:
"C:\Program Files\Cppcheck\cppcheck.exe" --help

の出力で以下とあります。

```
    --output-file=<file> Write results to file, rather than standard error.
```

Reply:
ローカルで `cppcheck --help` してオプションの存在を確認しました。

```cmd
--output-file=<file> Write results to file, rather than standard error.
```

問題なさそうですね、失礼しました。